emergency_stop_monitor:
  ros__parameters:
    start_with_emergency_stop: true

onboard_twist_mux:
  ros__parameters:
    topics:
      teleop:
        topic: teleop_mux_vel
        timeout: 0.5
        priority: 100
      waywise:
        topic: waywise_vel
        timeout: 0.5
        priority: 60
